### DISCUSSION TITLE
About Scxml File Import Xstate

### DISCUSSION THREAD

**Discussion by tycoding** [statelyai/xstate#4320] [Q&A]
Thanks Xstate, 

Why was the lib for parsing SCXML files removed in version 5? Does version 5 no longer support importing SCXML files directly into XState? Version 4 included lib/scxml.

**Comment by ChrisShank** [comment]
The source files for this code is still in the `v5` branch as far as i can tell. Maybe its a bundling issue or a problem with the packaje.json?

https://github.com/statelyai/xstate/blob/next/packages/core/src/scxml.ts#L583

**Reply by ChrisShank** [reply]
It doesn't look like this file is defined in the `exports` of the `package.json`

https://github.com/statelyai/xstate/blob/next/packages/core/package.json#L8
